src/main/sensors/gyro.c

```diff
@@ -69,6 +70,10 @@
 #include "hardware_revision.h"
 #endif
 
+#if (FLASH_SIZE > 128) && !defined(USE_GYRO_SPI_MPU6000)
```

Comment:
This will not work on targets that autodetect gyro (most of F4?)
`#if (FLASH_SIZE > 128) && defined(USE_GYRO_SPI_ICM20689)` should work ...